libnmstate/nm/plugin.py

```diff
@@ -165,12 +162,10 @@ def get_routes(self):
         return {Route.CONFIG: get_route_running_config(self._applied_configs)}
 
     def get_route_rules(self):
-        return {
-            RouteRule.CONFIG: (
-                get_ipv4_routing_rule_config(self.client)
-                + get_ipv6_routing_rule_config(self.client)
-            )
-        }
+        """
+        Nispor will provide running config of route rule from kernel.
+        """
+        return {}
```

Comment:
According to your review on #1410, we should still provinding routerule gathering support from NetworkManager. Am I missing something here?

Reply:
NetworkManager plugin only provide information unknown to kernel. For route rule, nispor already know full picture. NM plugin only required to apply config.